jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -79,6 +79,8 @@ async def open(self, type_path):
         self.saving_document = None
         self.id = str(uuid.uuid4())
         self.room_id = file_path
+        self.file_type = file_type
+        self.file_format = "text"
```

Comment:
Leftover?

Reply:
The `self.file_format = "text"` yes. `self.file_type = file_type` no